packages/itwin/breakdown-trees/src/Views/ClassificationsTree.tsx

```diff
@@ -46,7 +46,7 @@ export const ClassificationsTree: React.FC<ClassificationsTreeProps> = (props: C
     const functionIconMapperInner = props.additionalFunctionIconMapper ?? new TreeNodeFunctionIconInfoMapper(dataProvider);
     const optionItemsInner: OptionItemHandler[] = [];
     populateMapWithCommonMenuItems(CLASSIFICATIONS_TREE_NAME, functionIconMapperInner, dataProvider, classificationRules.id, props.eventHandlers);
-    const displayGuidHandlerInner = new GenericOptionItemHandler("Show Guids", BreakdownTrees.translate("contextMenu.showGuids"), "icon-label", () => { return props.displayGuids; }, props.setIsDisplayGuids);
+    const displayGuidHandlerInner = new GenericOptionItemHandler("Show Guids", BreakdownTrees.translate("contextMenu.showGuids"), "icon - label", () => { return props.displayGuids; }, props.setIsDisplayGuids);
```

Comment:
why?

Reply:
fixing that now, must have happened when i was doing a replace all by accident

Reply:
changed